stubs/pyOpenSSL/OpenSSL/SSL.pyi

```diff
@@ -1,26 +1,153 @@
 import socket
-from typing import Callable, Sequence
+from typing import Any, Callable, Sequence
 
-from OpenSSL.crypto import X509
+from OpenSSL.crypto import X509, PKey
 
+OPENSSL_VERSION_NUMBER: int
+SSLEAY_VERSION: int
+SSLEAY_CFLAGS: int
+SSLEAY_PLATFORM: int
+SSLEAY_DIR: int
+SSLEAY_BUILT_ON: int
+
+SENT_SHUTDOWN: int
+RECEIVED_SHUTDOWN: int
+
+SSLv2_METHOD: int
+SSLv3_METHOD: int
+SSLv23_METHOD: int
+TLSv1_METHOD: int
+TLSv1_1_METHOD: int
 TLSv1_2_METHOD: int
+TLS_METHOD: int
+TLS_SERVER_METHOD: int
+TLS_CLIENT_METHOD: int
+
+SSL3_VERSION: int
+TLS1_VERSION: int
+TLS1_1_VERSION: int
+TLS1_2_VERSION: int
+TLS1_3_VERSION: int
+
 OP_NO_SSLv2: int
 OP_NO_SSLv3: int
 OP_NO_TLSv1: int
 OP_NO_TLSv1_1: int
 OP_NO_TLSv1_2: int
 OP_NO_TLSv1_3: int
+
+MODE_RELEASE_BUFFERS: int
+
+OP_SINGLE_DH_USE: int
+OP_SINGLE_ECDH_USE: int
+OP_EPHEMERAL_RSA: int
+OP_MICROSOFT_SESS_ID_BUG: int
+OP_NETSCAPE_CHALLENGE_BUG: int
+OP_NETSCAPE_REUSE_CIPHER_CHANGE_BUG: int
+
+OP_SSLREF2_REUSE_CERT_TYPE_BUG: int
+OP_MICROSOFT_BIG_SSLV3_BUFFER: int
+OP_MSIE_SSLV2_RSA_PADDING: int
+OP_SSLEAY_080_CLIENT_DH_BUG: int
+OP_TLS_D5_BUG: int
+OP_TLS_BLOCK_PADDING_BUG: int
+OP_DONT_INSERT_EMPTY_FRAGMENTS: int
+OP_CIPHER_SERVER_PREFERENCE: int
+OP_TLS_ROLLBACK_BUG: int
+OP_PKCS1_CHECK_1: int
+OP_PKCS1_CHECK_2: int
+OP_NETSCAPE_CA_DN_BUG: int
+OP_NETSCAPE_DEMO_CIPHER_CHANGE_BUG: int
+
+OP_NO_COMPRESSION: int
+
+OP_NO_QUERY_MTU: int
+OP_COOKIE_EXCHANGE: int
+OP_NO_TICKET: int
+
+OP_ALL: int
+
 VERIFY_PEER: int
+VERIFY_FAIL_IF_NO_PEER_CERT: int
+VERIFY_CLIENT_ONCE: int
+VERIFY_NONE: int
+
+SESS_CACHE_OFF: int
+SESS_CACHE_CLIENT: int
+SESS_CACHE_SERVER: int
+SESS_CACHE_BOTH: int
+SESS_CACHE_NO_AUTO_CLEAR: int
+SESS_CACHE_NO_INTERNAL_LOOKUP: int
+SESS_CACHE_NO_INTERNAL_STORE: int
+SESS_CACHE_NO_INTERNAL: int
+
+SSL_ST_CONNECT: int
+SSL_ST_ACCEPT: int
+SSL_ST_MASK: int
+
+SSL_CB_LOOP: int
+SSL_CB_EXIT: int
+SSL_CB_READ: int
+SSL_CB_WRITE: int
+SSL_CB_ALERT: int
+SSL_CB_READ_ALERT: int
+SSL_CB_WRITE_ALERT: int
+SSL_CB_ACCEPT_LOOP: int
+SSL_CB_ACCEPT_EXIT: int
+SSL_CB_CONNECT_LOOP: int
+SSL_CB_CONNECT_EXIT: int
+SSL_CB_HANDSHAKE_START: int
+SSL_CB_HANDSHAKE_DONE: int
+
+NO_OVERLAPPING_PROTOCOLS = object()
+
+class Error(Exception): ...
+class WantReadError(Error): ...
+class WantWriteError(Error): ...
+class WantX509LookupError(Error): ...
+class ZeroReturnError(Error): ...
+class SysCallError(Error): ...
+
+def SSLeay_version(type: int) -> str: ...
+
+class Session: ...
```

Comment:
As Python 2 is still supported, this should be marked as a new-style class explicitly:

```suggestion
class Session(object): ...
```

(Same for the other classes below, but changing them in this PR is optional.)